consul/base.py

```diff
@@ -640,13 +646,38 @@ def delete(self, key, recurse=None, cas=None, token=None, dc=None):
             return self.agent.http.delete(
                 CB.json(), '/v1/kv/%s' % key, params=params)
 
+    class TXN(object):
```

Comment:
using `txn` is fine. I am only referring to the classname. Maybe just use `Txn` for the class name instead so it adheres to naming convention.